.github/workflows/ci.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        python-version: ["2.7", "3.6", "3.7", "3.8"]
+        django-version: ["1.11", "2.0", "2.2", "2.1", "3.0"]
+        es-dsl-version: ["6.4", "7.4"]
+        es-version: ["7.13.4"]
+
+        exclude:
+          - python-version: "2.7"
+            django-version: "2.0"
+          - python-version: "2.7"
+            django-version: "2.1"
+          - python-version: "2.7"
+            django-version: "2.2"
+          - python-version: "2.7"
+            django-version: "3.0"
+
+          - python-version: "3.8"
+            django-version: "1.11"
+          - python-version: "3.8"
+            django-version: "2.0"
+          - python-version: "3.8"
+            django-version: "2.1"
+
+    steps:
+      - name: Configure Sysctl Limits for Elasticsearch
+        run: |
+          sudo swapoff -a
+          sudo sysctl -w vm.swappiness=1
+          sudo sysctl -w fs.file-max=262144
```

Comment:
Do we really need swap?

Reply:
I believe 7GB ram would be enough to run the test suite!
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
This is recommended by elastic:

> You must also add the Configure sysctl limits step, otherwise Elasticsearch will not be able to boot.

Ref: https://github.com/elastic/elastic-github-actions/tree/master/elasticsearch#usage

Reply:
@safwanrahman I have removed this and it looks like everything is working as expected, not sure why it was recommended by elastic, maybe there was some edge case before as the README was last updated more than a year ago.

Output: https://github.com/saadmk11/django-elasticsearch-dsl/actions/runs/1119509026